src/views/TransferOrderDetail.vue

```diff
@@ -65,7 +65,7 @@
                 </div>
                 <div class="order-metadata">
                   <ion-label>
-                    {{ getShipmentMethodDesc(shipment.shipmentMethodTypeId) }}
+                    {{ shipment.routeSegShipmentMethodDescription }}
```

Comment:
We should add a fallback to display shipmentMethodId in case description is not returned from the api